common_utils/utils.py

```diff
@@ -31,3 +57,147 @@ def get_original_client_ip():
             client_ip = request.META.get("REMOTE_ADDR")
 
     return client_ip
+
+
+def generate_email_template(template_source_path, save=False):
+    with open(template_source_path, "r") as template_source_file:
+        template_source = template_source_file.read()
+        template = Environment(
+            loader=FileSystemLoader(EMAIL_TEMPLATES_PATH)
+        ).from_string(template_source)
+
+        try:
+            rendered = template.render(
+                image_location=settings.EMAIL_TEMPLATE_IMAGE_SOURCE
+            )
+
+            return rendered
+
+        except Exception:
+            print("Unexpected error:", sys.exc_info()[0])
```

Comment:
is this good enough in this context?

Reply:
Depending on how the logging is configured, the following might automagically also print the exception.
```python
except Exception:
    logger.exception("Error in generating the templates.")
```